assets/admin_themes/Sea_Green/modal/modal.scss

```
@@ -1,53 +1,92 @@
-            background-color : $g-200;
+  .btn-close {
+    align-self: flex-start;
+    margin: 0 -0.75rem -0.75rem auto;
+    color: $g-900;
+    width: 0.75em;
+    height: 0.75em;
+    opacity: 1;
+    font-size: $btn-font-size-sm;
+    &:focus {
+      box-shadow: none;
+      outline: 2px solid currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)